Standards/scs-0100-v3-flavor-naming.md

```diff
@@ -258,27 +258,6 @@ You must not offer flavors with the `SCS-` prefix which do not follow this namin
 You must not extend the SCS naming scheme with your own extensions; you are encouraged however
 to suggest extensions that we can discuss and add to the official scheme.
 
-## Conformance Tests
```

Comment:
Additionally, we should extend [standard SCS standards](https://github.com/SovereignCloudStack/standards/blob/issue/352/Standards/scs-0001-v2-sovereign-cloud-standards.md) and mention implementation and testing notes. We could do this as part of #352, e.g. in section _Conformance Tests_. It should be pointed out, that each standard should EITHER has a section called "Conformance Tests_ OR a separate document for implementation and testing notes.

Reply:
I meant to respond to @artificial-intelligence here, but I seem to have missed it. Having these (hard-coded) references between the documents doesn't sit well with me. I think what we can do is adjust the auto-generated tables in the docs pages so that they list supplementary documents with the corresponding standard, so these supplements can't be missed. I fully agree with extending scs-0001 though.